.github/workflows/add-flaky-test-label.yml

```diff
@@ -29,6 +29,14 @@ jobs:
         if: ${{ hashFiles('**/pr-number') != '' }}
         run: |
           gh pr edit "$(cat pr-number)" --add-label 'triage/flaky-test'
+      - name: Install JDK 17 for JBang
```

Comment:
I would go with 21, it's the latest lts.

Reply:
alright